ebpf-profiling.graphqls

```diff
@@ -30,10 +30,45 @@ input EBPFProfilingTaskFixedTimeCreationRequest {
     targetType: EBPFProfilingTargetType!
 }
 
+input EBPFNetworkDataCollectingSettings {
+    # Require to collect the complete request
+    requireCompleteRequest: Boolean!
+    # The max size of request context. The unit is byte.
+    # Collect the whole request header and body if this is not set.
+    maxRequestSize: Int
+
+    # Require to collect the complete response
+    requireCompleteResponse: Boolean!
+    # The max size of response context. The unit is byte.
+    # Collect the whole response header and body if this is not set.
+    maxResponseSize: Int
+}
+
+input EBPFNetworkSamplingRule {
+    # The match pattern for HTTP request. This is HTTP URI-oriented.
+    # matches all requests if not set
+    uriRegex: String
+
+    # the minimal request duration to activate the network data(HTTP request/response raw data) sampling.
+    # Collecting requests without minimal request duration
+    minDuration: Int
+    # Collecting requests when the response code is 400-499
+    when4xx: Boolean!
+    # Collecting requests when the response code is 500-599
+    when5xx: Boolean!
```

Comment:
`minDuration` condition could work when `when5xx` or `when4xx` matched. If neither of them is set, `minDuration` works on its own.

Reply:
If the `when5xx` is matched, but the duration of the response is small than `minDuration`, should be sampling?

Reply:
No, we should not.

Reply:
Sure, let me verify the format. Once the format verifies success, I think it could be merged.